src/signals/incident/components/form/PlainText/index.js

```diff
@@ -1,19 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import isString from 'lodash.isstring';
 import get from 'lodash.get';
+import { isAuthenticated } from 'shared/services/auth/auth';
 
+import { themeColor } from '@datapunt/asc-ui';
 import mapDynamicFields from '../../../services/map-dynamic-fields';
 import './style.scss';
 
-function renderText(value, parent) {
+// This control will disable all the links when the user is not authenticated.
+const Span = styled.span`
+  & a {
+    ${({ authenticated }) => authenticated
+    ? css`
+            color: ${themeColor('primary')};
+            font-weight: bold;
+          `
+    : css`
+            pointer-events: none;
+            cursor: default;
+            text-decoration: none;
+            color: inherit;
+          `
+}}
+`;
+
+const renderText = (value, parent) => {
   if (React.isValidElement(value)) {
     return value;
   }
 
-  return mapDynamicFields(value, { incident: get(parent, 'meta.incidentContainer.incident') });
-}
+  const text = mapDynamicFields(value, { incident: get(parent, 'meta.incidentContainer.incident') });
+  return (
+    <Span
+      authenticated={isAuthenticated()}
+      // eslint-disable-next-line react/no-danger
+      dangerouslySetInnerHTML={{
+        __html: text,
+      }}
+    />
+  );
+};
```

Comment:
Can you add a test case that tests the difference between being authenticated and not authenticated? Now, the snapshot only contains not being authenticated.